src/main/java/exercise9th/mybatisdemo/NameController.java

```diff
@@ -0,0 +1,25 @@
+package exercise9th.mybatisdemo;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class NameController {
+
+    private NameMapper nameMapper;
```

Comment:
それからfinalをつけるとよいです。
```suggestion
    private final NameMapper nameMapper;
```

理由については
コンストラクタインジェクションの長所を調べるとよいです。
キーワードとして`Immutable`という言葉があります。

Reply:
コンストラクタインジェクションの長所を調べました。
・リクエスト（＝スレッド）ごとにインスタンスが生成され、スレッド間でインスタンス変数が共有されなくなる。
・フィールドインジェクションと異なり、フィールドにfinal修飾子を使える。
・final修飾子により、コンストラクタ以外では値の変更を許容せず、マルチスレッドではImmutableオブジェクトのスレッドセーフを保証する。（他スレッドからは、必ずコンストラクタによる初期化完了後の値が見える）

Reply:
> コンストラクタインジェクションの長所を調べました。

内容確認しました！ばっちりです！ 😎